orga/app/routes/join.js

```diff
@@ -7,7 +7,12 @@ export default Route.extend(UnauthenticatedRouteMixin, {
     return this.store.queryRecord('organization-invitation', {
       invitationId: params.invitationId,
       code: params.code
-    }).catch(() => {
+    }).catch((errorResponse) => {
+      errorResponse.errors.forEach((error) => {
+        if (error.status === '421') {
+          this.replaceWith('login', { queryParams: { hasInvitationError: true } });
+        }
+      });
       this.replaceWith('login');
```

Comment:
du coup est-ce qu'on a toujours besoin de cette ligne ?

btw, là si ya une autre erreur unexpected, on ne la loggue jamais. Je crois qu'il faut faire un `return this.send('error')`

Reply:
Cette ligne est là pour garder le fonctionnement précédent (qui n'est pas optimal car il n'y a pas une bonne gestion d'erreur). Par manque de maîtrise d'Ember, je préfère laisser tel quel pour le moment

Reply:
Attention ce code peut faire plusieurs `replaceWith` (s'il y a plusieurs erreurs 421, plus le `replaceWith('login');` de la fin), c'est vraiment pas recommandé pour avoir un comportement prévisible !